src/app/(app)/api/stripe/webhooks/route.ts

```
@@ -0,0 +1,99 @@
+      { status: 400 }
+    );
+  }
+
+  const permittedEvents: string[] = ["checkout.session.completed"];
+
+  const payload = await getPayload({
+    config,
+  });
+
+  if (permittedEvents.includes(event.type)) {
+    let data;
```

Biome (2.1.2)

[error] This variable implicitly has the any type.

Variable declarations without type annotation and initialization implicitly have the any type. Declare a type or initialize the variable with some value.

(lint/suspicious/noImplicitAnyLet)